src/java.net.http/share/classes/java/net/http/HttpRequest.java

```diff
@@ -292,6 +292,10 @@ default Builder HEAD() {
         /**
          * Builds and returns an {@link HttpRequest}.
          *
+         * @implSpec Returns a new {@code HttpRequest} each time it is
+         * invoked. Once built, the HttpRequest is immutable and can be
+         * sent multiple times.
```

Comment:
Looks fine to me. Maybe "This method returns ..." would more appropriate for an `@implSpec`. Also the second mention of `HttpRequest` should also have `{@code }` around it.